### PR TITLE
0.9.1 release

### PR DESCRIPTION
**Pull request by barrywhart**
### Pull Request checklist
- [x] Please confirm you have completed any of the necessary steps below.

- Included test cases to demonstrate any code changes, which may be one or more of the following:
  - `.yml` rule test cases in `test/fixtures/rules/std_rule_cases`.
  - `.sql`/`.yml` parser test cases in `test/fixtures/dialects` (note YML files can be auto generated with `tox -e generate-fixture-yml`).
  - Full autofix test cases in `test/fixtures/linter/autofix`.
  - Other.
- Added appropriate documentation for the change.
- Created GitHub issues for any relevant followup/future enhancements if appropriate.

### CODE REVIEW

**Comment by jpy-git** [comment]
@barrywhart I like the new section for calling out new contributors 😊

Think you need to update both the sqlfluff and sqlfluff-templater-dbt setup.CFC files with the new version

**Comment by jpy-git** [comment]
Also there are a couple PRs open atm so tag the contributors to let them know 😊

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,116 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+
+## Highlights
+
+* Fix dbt 1.0.0 connection issue
+* Fix some SQL corruption issues with templated code
+* New components to simplify creating rules
+* Remove support for Python 3.6
+
+## What's Changed
+* fix delimited identifier parsing for spark3 [#2111](https://github.com/sqlfluff/sqlfluff/pull/2111) [@mcannamela](https://github.com/mcannamela)
```
Comment (tunetheweb) [inline CHANGELOG.md:22]: Nit (I've updated the title in PR btw):

```suggestion
* Fix delimited identifier parsing for spark3 [#2111](https://github.com/sqlfluff/sqlfluff/pull/2111) [@mcannamela](https://github.com/mcannamela)
```

**Comment by barrywhart** [comment]
@jpy-git: `.cfg` files updated, and I have tagged the two PR authors in Slack. Thanks!

**Review by NiallRees — APPROVED** [review]

**Review by tunetheweb — APPROVED** [review]
One minor nit from a quick review.

Not sure if you wanna split into "enhancements" and "bug fixes"? Not sure it's worth the effort to be honest.

**Comment by barrywhart** [comment]
I'm okay without splitting bugs and enhancements.

**Comment by jpy-git** [comment]
I'll pick up @R7L208's PR #2256 for review.

If someone else can pick up @chwiese's PR #2250 review that would be great 👍

**Comment by barrywhart** [comment]
I got #2250.

**Comment by jpy-git** [comment]
Okay #2250 and #2256 are both merged now! 🎉 

@barrywhart could you update the CHANGELOG to include these 2 PRs and then we're good to go ahead with the release 👍

**Comment by barrywhart** [comment]
@jpy-git: 👍 I'll probably do the release before bed tonight (currently 5:30pm here).

**Review by jpy-git — APPROVED** [review]

**Pull request merged**